src/ts/uimanager.ts

```diff
@@ -456,8 +455,8 @@ export class UIManager {
    * be found and thus not removed.
    */
   removeTimelineMarker(timelineMarker: TimelineMarker): boolean {
-    if (ArrayUtils.remove(this.config.metadata.markers, timelineMarker) === timelineMarker) {
-      this.config.events.onUpdated.dispatch(this);
+    if (ArrayUtils.remove(this.uiconfig.metadata.markers, timelineMarker) === timelineMarker) {
```

Comment:
Again, `mergedConfig`? I don't think the unprocessed `uiconfig` should evner be used, that's why I originally didnt even store it in a field. Only the processed config should be used.